go/vt/vtgate/engine/ordered_aggregate_test.go

```diff
@@ -652,13 +653,15 @@ func TestMerge(t *testing.T) {
 		"1|3|2.8|2|bc",
 	)
 
-	merged, _, err := oa.merge(fields, r.Rows[0], r.Rows[1], nil)
+	// TODO(king-11) make collation aware
```

Comment:
shouldn't this work if we are adding collations to ordered_aggregate?

Reply:
for this test we don't need to pass in collation I added `TODO` by mistake this works as is supposed to.

Reply:
made the change removed `TODO`